docs/tutorials/projects.md

```diff
@@ -0,0 +1,348 @@
+
+<div class="terminal">
+```bash
+$ prefect deploy ./api_flow.py:call_api \
+    -n my-first-deployment \
+    -p local-work
+```
+</div>
+
+This command will create a new deployment for your `"Call API"` flow with the name `"my-first-deployment"` that is attached to the `local-work` work pool.
+
+Note that Prefect has automatically done a few things for you:
+
+- registered the existence of this flow [with your local project](/concepts/projects/#the-prefect-directory)
+- created a description for this deployment based on the docstring of your flow function
+- parsed the parameter schema for this flow function in order to expose an API for running this flow
+
+You can customize all of this either by [manually editing `deployment.yaml`](/concepts/projects/#the-deployment-yaml-file) or by providing more flags to the `prefect deploy` CLI command; CLI inputs will always be prioritized over hard-coded values in your deployment's YAML file.
+
+Let's create two ad-hoc runs for this deployment and confirm things are healthy:
+<div class="terminal">
+```bash
+$ prefect deployment run 'Call API/my-first-deployment'
+$ prefect deployment run 'Call API/my-first-deployment' \
+    --param url=https://cat-fact.herokuapp.com/facts/
+```
+</div>
+
+You should now be able to monitor and confirm these runs were created and ran via your [server UI](/ui).
+
+!!! tip "Flow registration"
+    `prefect deploy` will automatically register your flow with your local project; you can register flows yourself explicitly with the `prefect project register-flow` command:
+    <div class="terminal">
+    ```bash
+    $ prefect project register-flow ./api_flow.py:call_api
+    ```
+    </div>
+
+    This pre-registration allows you to deploy based on name instead of entrypoint path:
+    <div class="terminal">
+    ```bash
+    $ prefect deploy -f 'Call API' \
+        -n my-first-deployment \
+        -p local-work
+    ```
+    </div>
+
+### Git-based deployment
+
+In this example, we'll initialize a project from [a pre-built GitHub repository](https://github.com/PrefectHQ/hello-projects) and see how it is automatically portable across machines.
+
+We start by cloning the remote repository and initializing a project within the root of the repo directory:
+
+<div class="terminal">
+```bash
+$ git clone https://github.com/PrefectHQ/hello-projects
+$ cd hello-projects
+$ prefect project init --recipe git
+```
+</div>
+
+We can now proceed with the same steps as above to create a new deployment:
+
+<div class="terminal">
+```bash
+$ prefect deploy -f 'log-flow' \
+    -n my-git-deployment \
+    -p local-work
+```
+</div>
+
+Notice that we were able to deploy based on flow name alone; this is because the repository owner [pre-registered the `log-flow` for us](https://github.com/PrefectHQ/hello-projects/blob/main/.prefect/flows.json).  Alternatively, we could run `prefect deploy ./flows/log_flow.py:log_flow` if we knew the full entrypoint path.
```

Comment:
```suggestion
Notice that we were able to deploy based on flow name alone; this is because the repository owner [pre-registered the `log-flow` for us](https://github.com/PrefectHQ/hello-projects/blob/main/.prefect/flows.json).  Alternatively, if we knew the full entrypoint path, we could run `prefect deploy ./flows/log_flow.py:log_flow`.
```